File: src/targets/node/axios/fixtures/text-plain.js
<<<<<<< HEAD
const axios = require('axios');

const options = {
  method: 'POST',
  url: 'https://httpbin.org/anything',
=======
const axios = require('axios').default;

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  headers: {'content-type': 'text/plain'},
  data: 'Hello World'
};

<<<<<<< HEAD
axios
  .request(options)
  .then(function (response) {
    console.log(response.data);
  })
  .catch(function (error) {
    console.error(error);
  });
=======
try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}
>>>>>>> upstream/master

File: src/targets/swift/urlsession/fixtures/text-plain.swift
import Foundation
#if canImport(FoundationNetworking)
  import FoundationNetworking
#endif

let headers = ["content-type": "text/plain"]

let postData = Data("Hello World".utf8)

let url = URL(string: "https://httpbin.org/anything")!
var request = URLRequest(url: url)
request.httpMethod = "POST"
request.timeoutInterval = 10
request.allHTTPHeaderFields = headers
request.httpBody = postData

let (data, response) = try await URLSession.shared.data(for: request)
print(String(decoding: data, as: UTF8.self))